app/controllers/api/v1/mentors/sessions_controller.rb

```diff
@@ -0,0 +1,61 @@
+class Api::V1::Mentors::SessionsController < Devise::SessionsController
```

Comment:
I could observe you're using two `technologies_controller.rb`, one for mentors and the other for users, good job because this works but, why not add only one session controller using for both of them, maybe we can use only the user table, and inside this assign who is a mentor, not create an entire table for other users. This could result in more work on the code in the future and a lot of headaches.

Reply:
Hey @vichuge, I think you misunderstood this `technologies_controller`. This is only meant for mentors. We know how to implement two users in one model. we will implement it later. If we try it now, it will disturb the whole project structure, and since we don't have much time, we can't implement it at this moment. We really appreciate your comments. 

Thanks.

Reply:
Sure, no problem. Remember:

`
In this project, code reviews are not the final evaluation of students's projects. Their purpose is to help students to improve their project BEFORE the final presentation. Please take that into account while communicating with students - you can (and you should) suggest as many improvements you can but students can use your tips or ignore them.
`